pf4j/src/main/java/org/pf4j/AbstractExtensionFinder.java

```diff
@@ -174,7 +174,7 @@ public AbstractExtensionFinder(PluginManager pluginManager) {
         List<ExtensionWrapper> result = new ArrayList<>();
 
         Set<String> classNames = findClassNames(pluginId);
-        if (classNames.isEmpty()) {
+        if (classNames == null || classNames.isEmpty()) {
```

Comment:
Also, please remove this modification (it's enough the initial condition `classNames.isEmpty()`).

Reply:
https://github.com/pf4j/pf4j/blob/cf2efbd3a00219c5c43df7379bc44b28373dc298/pf4j/src/main/java/org/pf4j/AbstractExtensionFinder.java#L86

And this?

Reply:
> https://github.com/pf4j/pf4j/blob/cf2efbd3a00219c5c43df7379bc44b28373dc298/pf4j/src/main/java/org/pf4j/AbstractExtensionFinder.java#L86
> 
> And this?

Good catch. Yes please